bin/iotagent-json

```diff
@@ -42,13 +39,7 @@ function start() {
 
     config.iota.iotaVersion = info.version;
 
-    iotAgent.start(config, function (error) {
```

Comment:
if `iotAgent.startServer()` returned a callback, it wouldn't be necessary to delete this useful debug.